src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -144,7 +145,9 @@ function ExamSettings() {
   );
 }
 
-ExamSettings.propTypes = {};
+ExamSettings.propTypes = {
+  courseKey: PropTypes.string.isRequired,
```

Comment:
I think it would be better for this prop to also be `courseId`. The reason I bring this is up is that I recently had confusion around the difference between `course key` and `course ID`.

My understanding is a course ID is a string representation of a course key. A course key refers to a `CourseKey` object as defined by our [Opaque Keys](https://github.com/edx/edx-platform/wiki/Opaque-Keys-(Locators)) system. For example, a string course ID is what you would pass as a parameter to the `CourseKey.from_string` method to generate a `CourseKey`.

If we can maintain that distinction here, I think it would be a good idea, as it would reduce confusion. Let me know what you think!

Reply:
That definitely makes sense, I have also been wondering if the two were interchangeable. Thank you for that explanation!

Reply:
👍  for attempting to maintain this distinction